src/main/java/de/ipvs/as/mbp/domain/access_control/ACDoubleAccuracyEffect.java

```diff
@@ -49,8 +49,12 @@ public ACDoubleAccuracyEffect(String name, String description, Map<String, Strin
 	
 	@Override
 	public ValueLog apply(ValueLog valueLog) {
-		return valueLog.setValue(apply(valueLog.getValue()));
+		// TODO NOT SUPPORTED AT THE MOMENT
```

Comment:
It would be relatively easy to implement a solution for the complex data there (iterate over all leaf nodes of the type double and modify them using the DocumentReader from the utils package). However, the apply method was never executed in my tests which is why it seems to me like the ACEffects do not work properly at all (even on the main branch).